modules/ocf_puppet/files/write-otp-config

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: write-otp-config <username> <yubikey public ID>"
+    exit 1
+fi
+
+username="$1"
+yubikeyID="$2"
+
+if grep -q "^$username:" /opt/puppet/shares/private/otp/yubikeys; then
+    echo "$username already registered. Appending yubikey, not reconfiguring google authenticator."
+    sed -i "/^$username:/ s/$/:$yubikeyID/" /opt/puppet/shares/private/otp/yubikeys
+    exit 0
+fi
+
+echo "$username:$yubikeyID" >> /opt/puppet/shares/private/otp/yubikeys
```

Comment:
does this add you twice? if so, shouldn't this be in a else block

Reply:
see line 16